tests/formatter/test_tools.py

```diff
@@ -48,3 +48,40 @@ def test_tools_ordering(fmt: Fmt) -> None:
     a = 0
         """
     fmt(fmt_tools, content, expected)
+
+
+def test_sub_table_newline(fmt: Fmt) -> None:
+    content = """
+    [tool.mypy]
+    a = 0
+
```

Comment:
Can we remove this newline too here 😅

Reply:
Oh, I see why you did not do it. 🤔 If you remove this line this patch will not add the newline in.